feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/presentation/unlock/confirm/hints/ConfirmGovernanceUnlockHintsMixin.kt

```diff
@@ -35,7 +35,7 @@ private class ConfirmGovernanceUnlockHintsMixin(
 ) : HintsMixin, WithCoroutineScopeExtensions by WithCoroutineScopeExtensions(scope) {
 
     override val hintsFlow: Flow<List<CharSequence>> = remainsLockedInfoFlow.map { remainsLockedInfo ->
-        if (remainsLockedInfo != null) {
+        if (remainsLockedInfo != null && remainsLockedInfo.lockedInIds.isNotEmpty()) {
```

Comment:
Small fix for cornercase when transferable balance doesn't change due to reserved balance overlapping with locks